src/bookshelf/index.ts

```diff
@@ -30,7 +30,7 @@ export default class Bookshelf implements Mapper {
     let linkOpts: LinkOpts = { baseUrl: this.baseUrl, type, pag: bookOpts.pagination };
 
     // Set default values for the options
-    bookOpts = assign({relations: true, enableLinks: true}, bookOpts);
+    bookOpts = assign({relations: { included: true }, enableLinks: true}, bookOpts);
```

Comment:
Also, doing this may make the `included?` in `RelationOpts` not be optional, right?